src/scripts/achievements/AchievementTracker.ts

```diff
@@ -0,0 +1,51 @@
+class AchievementTracker implements Feature {
+    name = 'AchievementTracker';
+    saveKey = 'achievementTracker';
+    trackedAchievement: KnockoutObservable<Achievement>;
+
+    defaults = {
+        'trackedAchievement': null
+    };
+
+    constructor() {
+        this.trackedAchievement = ko.observable(this.defaults.trackedAchievement)
+    }
+
+    initialize(): void {
+
+    }
+
+    canAccess(): boolean {
+        return App.game.party.caughtPokemon.length >= 110;
```

Comment:
I picked this threshold because:
* I didn't think we should dump this on a player right at the start
* I wouldn't imagine players would seriously start going for achievements before they hit the wall near the end of Kanto anyway, which is around ~120/130 caught Pokemon.

Happy to adjust/remove this if people feel it's too restrictive.